pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -50,6 +50,11 @@ cdef extern from "../src/datetime/np_datetime.h":
         PANDAS_FR_fs
         PANDAS_FR_as
 
+    int days_per_month_table[2][12]
+    int dayofweek(int y, int m, int d) nogil
+    int is_leapyear(int64_t year) nogil
```

Comment:
There are a couple of modules that use one or more of these three items but _dont_ need the rest of the relatively heavy-weight src/datetime/np_datetime dependency.  Ideally I'd prefer for these to be implemented in a dependency-free cython module.

Reply:
sure these are like dateutils